src/index.ts

```diff
@@ -0,0 +1,10 @@
+import { HubApiModel } from "./model";
+import { version } from '../package.json';
+
+export = {
+    name: 'koop-provider-hub-search-ogc',
+    type: 'provider',
+    disableIdParam: false,
```

Comment:
What's the point of this flag?

Reply:
`disableIdParam: false` allows default `id` parameter from routes. For example https://koop-plugin-endpoint/:id/items where `id` is being used as collection items.